migrations/Version202012161236262141_taoItems.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
```

Comment:
Missing doc

Reply:
but this being a migration i do see it as absolutely required